src/components/Page/Base.js

```diff
@@ -1,13 +1,17 @@
 import { NextSeo } from "next-seo";
+import dynamic from "next/dynamic";
 import PropTypes from "prop-types";
-import React from "react";
+import React, { useState } from "react";
 
 import Footer from "@/pesayetu/components/Footer";
 import Navigation from "@/pesayetu/components/Navigation";
-import { navigationArgs } from "@/pesayetu/config";
+import { navigationArgs, tour } from "@/pesayetu/config";
 import getFooterMenu from "@/pesayetu/functions/menus/getFooterMenu";
 import getNavigationMenu from "@/pesayetu/functions/menus/getNavigationMenu";
 
+const Tour = dynamic(() => import("@/pesayetu/components/HURUmap/Tour"), {
+  ssr: false,
+});
```

Comment:
Why do we have this in _Base_ page?

Reply:
I wanted it close to the navigation. Also, the Plugin doesn't have support for SSR yet. @kilemensi

Reply:
I have wrapped it in an if statement, to check if its in explore mode